python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -16,11 +16,13 @@
 TENSOR_TRANSPORT_TO_COLLECTIVE_BACKEND = {
     TensorTransportEnum.NCCL: Backend.NCCL,
     TensorTransportEnum.GLOO: Backend.TORCH_GLOO,
+    TensorTransportEnum.NIXL: Backend.NIXL,
 }
 
 COLLECTIVE_BACKEND_TO_TORCH_DEVICE = {
     Backend.NCCL: torch.device("cuda"),
     Backend.TORCH_GLOO: torch.device("cpu"),
+    Backend.NIXL: torch.device("cuda"),
```

Comment:
This is okay for now but doesn't NIXL actually support both CPU and CUDA tensors? Let's add a TODO.